internal/core/metadata/operators/device/get_test.go

```diff
@@ -15,25 +15,29 @@
 package device
 
 import (
-	"github.com/pkg/errors"
 	"testing"
 
-	"github.com/edgexfoundry/edgex-go/internal/core/metadata/operators/device/mocks"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/config"
+	"github.com/pkg/errors"
```

Comment:
This should be last in the ordering, alphabetizing everything after github.com. Also edgex-go comes before go-mod-core-contracts.